README.md

```diff
@@ -43,3 +43,10 @@ docker-compose -f docker-compose.ci.yml run ci ./project_tests.sh
 ```
 
 `-u` is needed to write to `vendor/`. Currently Composer prints some warning when git-cloning due to the user not being in /etc/passwd.
+
+## Journey of a request to the annotations api
+1. Query received via the `annotations` api.
+1. Retrieve annotation listing from `hypothes.is/api` based on query parameters.
+1. Denormalize the annotations into an object model.
+1. Normalize the annotations to prepare the response which conforms to the eLife json schema. [HTMLPurifier](http://htmlpurifier.org/) is used to sanitise content values. We rely on [CommonMark](http://commonmark.thephpleague.com/) to step through the Markdown content and prepare the structure that we need.
```

Comment:
would invert the two sentences: first we step through, and while stepping through we can sanitise?